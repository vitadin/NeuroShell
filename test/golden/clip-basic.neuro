%% Test basic clipboard functionality
\clip Hello World!
\clip Testing with variable interpolation: ${@user}
\set[message="Important data"]
\clip The message is: ${message}
\clip