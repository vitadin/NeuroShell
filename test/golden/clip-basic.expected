Copied 12 characters to clipboard
Copied 45 characters to clipboard
Setting message = Important data
Copied 30 characters to clipboard
Warning: No content specified. Clipboard unchanged.